### PR TITLE
[TEVA-3846] visual testing jobseeker authentication

### PR DESCRIPTION
**Pull request by alexbowen**
~~all images gitignored pending s3 integration
github workflow disabled pending same~~

~~also will rework this a little bit in near future so now dependant on review app deploy~~

### CODE REVIEW

File: config/ab_tests.yml [hunk 1]
```diff
@@ -43,3 +43,7 @@ test:
   2022_01_subjects_filters_test:
     default: 1
     subjects_filters: 0
+visual:
+  2022_01_homepage_quick_links_test:
+    without_quick_links: 1
+    with_quick_links: 1
```
Comment (josephhull676) [inline config/ab_tests.yml:49]: So in the visual tests we want an equal chance of either variant being assigned?

File: config/backstop/backstop.config.js [hunk 2]
```diff
@@ -1,43 +1,44 @@
...
+scenarios = scenarios.map((scenario) => {
+  return {...scenario, url: `${TEST_ENV_URL}${scenario.url}`}
+})
+
 module.exports = {
   "id": "teacher_vacancies",
   "viewports": [
     {
       "label": "phone",
       "width": 320,
```
Comment (josephhull676) [inline config/backstop/backstop.config.js:25]: I can't remember. Did we say we'd change this?

File: config/backstop/engine_scripts/playwright/loadCookies.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+      "httpOnly": false,
+      "secure": false,
+      "sameSite": "Lax"
+    })
+  }
+
+  // Add cookies to browser
+  browserContext.addCookies(cookies);
+
+  // console.log('Cookie state restored with:', JSON.stringify(cookies, null, 2));
```
Comment (josephhull676) [inline config/backstop/engine_scripts/playwright/loadCookies.js:28]: Remove?

File: config/backstop/engine_scripts/playwright/overrideCSS.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
+/**
+ * OVERRIDE CSS
+ * Apply this CSS to the loaded page, as a way to override styles.
+ *
+ * Use this in an onReady script E.G.
+  ```
+  module.exports = async function(page, scenario) {
+    await require('./overrideCSS')(page, scenario);
+  }
```
Comment (josephhull676) [inline config/backstop/engine_scripts/playwright/overrideCSS.js:9]: Remove?

**Review by josephhull676 — APPROVED** [review]
Perhaps something in the README on how to run the tests and how to create the reference images if necessary.

File: config/ab_tests.yml (hunk 1 above)
Comment (josephhull676) [reply config/ab_tests.yml:49]: I think the subjects filters test will need to be covered too!

Comment (josephhull676) [reply config/ab_tests.yml:49]: You'll probably want to fix the variant used in these tests by setting one variant to 1 and the other to 0

File: config/backstop/backstop.config.js (hunk 2 above)
Comment (alexbowen) [reply config/backstop/backstop.config.js:25]: we did but its not a task for now

File: config/backstop/engine_scripts/playwright/overrideCSS.js (hunk 4 above)
Comment (alexbowen) [reply config/backstop/engine_scripts/playwright/overrideCSS.js:9]: this is default/examples that backstop provides, ill leave this commented for now but any commenting ive done of my code ill remove

File: config/backstop/engine_scripts/playwright/loadCookies.js (hunk 3 above)
Comment (alexbowen) [reply config/backstop/engine_scripts/playwright/loadCookies.js:28]: like this

File: config/ab_tests.yml (hunk 1 above)
Comment (alexbowen) [reply config/ab_tests.yml]: wil explain what ive done here, its much better now

**Pull request merged**